api/v1beta1/agentserviceconfig_types.go

```diff
@@ -50,6 +50,13 @@ type MustGatherImage struct {
 	Url string `json:"url"`
 }
 
+type ImageServiceOptions struct {
+	// InsecureSkipVerify toggles whether or not the image service verifies
+	// certificates when downloading iso images from a server.
```

Comment:
nit: this may not be limited to ISO images.

But it would be good to be explicit about in which circumstances TLS verification is skipped. I assume this does not affect communication with assisted-service for example?

"InsecureSkipVerify toggles whether or not the image service verifies TLS certificates when downloading image artifacts from the sources configured in OSImages..." or something like that.